src/helpers.ts

```diff
@@ -6,6 +6,44 @@ export const ZeroAddress = "0x0000000000000000000000000000000000000000";
 export const EthAddress = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
 export const WethAddress = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2";
 
+export const PickleJars = [
```

Comment:
Not sure if this is the best approach. As far as I can tell we have 3 options: 
1. Use this and update it whenever there is a new pickle jar we need to support
2. Only include the `pSLP yvBoost-ETH` jar and maintain a small list of jars we will support
3. Cache the response of the pickle jars api so it's not fetched very often (https://stkpowy01i.execute-api.us-west-1.amazonaws.com/prod/protocol/pools)